testing/main_integration_test.go

```diff
@@ -90,7 +90,7 @@ func TestSetup(t *testing.T) {
 }
 
 func installPackage(t *testing.T, p string) {
-	req, err := http.NewRequest("POST", "http://elastic:changeme@localhost:5601/api/fleet/epm/packages/"+p, nil)
+	req, err := http.NewRequest("POST", "http://elastic:changeme@localhost:8220/api/fleet/epm/packages/"+p, nil)
```

Comment:
This is Kibana API, so it should stay as is (5601).

Reply:
Thanks, but I'm curious: how do you know which APIs are Kibana APIs and which ones are Fleet Server ones? Do you just happen to know this now because of trial-and-error and learning it the hard way or is there some place where these are documented?

Reply:
I'm afraid it's the first one: trial-and-error and learning it the hard way.

I also consider the elastic-package repo as the source of truth. We have logic able to install packages there and it works :)